src/containers/SearchResults.js

```diff
@@ -0,0 +1,27 @@
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { withTranslation } from 'react-i18next';
+import { withStyles } from '@material-ui/core/styles';
+import { withPlugins } from '../extend/withPlugins';
+import { SearchResults } from '../components/SearchResults';
+import {
+  getSearchHitsForManifest,
+} from '../state/selectors';
+
+/**
+ * mapStateToProps - used to hook up connect to state
+ * @memberof SearchResult
+ * @private
+ */
+const mapStateToProps = (state, { windowId }) => ({
+  searchHits: getSearchHitsForManifest(state, { windowId }),
```

Comment:
Sorry, I may have missed the boat on the data shape, but `getSearchHitsForManifest` probably needs to get keyed on the companion window id too, and that'll need to trickle down all the way to the redux state?

Reply:
Possibly? Is that for garbage collection? I was kind of just kicking the can down until the `clear` button got implemented.

Reply:
Happy to chat about this.

Reply:
Yeah, I guess that's a good question.  Not sure if you have chatted about this already, but this may be a question for @jvine / @ggeisler (e.g. when I pop out the search sidebar to a companion window on the right, then open up the search again in the sidebar, should my previous search be initialized, or should I be getting a blank search box to initialize a new search)

Reply:
Spinning this out to a new issue https://github.com/ProjectMirador/mirador/issues/2685 . I'm 👍 to this, but think it could happen in concert w/ other issues

Reply:
And I think it would mean that the CS behavior would be different than the annotations panel behavior, no?

Reply:
Yeah, it has pretty large implications for differing behavior.  I think we can ship this as is assuming that is the behavior we want (and seeing that #2687 is ready).  I _assume_ this is ultimately what we want.

Reply:
:shipit: